tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -1076,6 +1076,86 @@ StatusOr<bool> IsIdentityDrivingConstsInLoop(Node* node) {
   return true;
 }
 
+const absl::flat_hash_map<string, std::vector<string>> whitelist_table = {
```

Comment:
What invariants must hold for this list? Or phrased differently, at what points does this list to be updated? Is it meant to be complete, i.e. all point-wise operations supported by XLA:GPU are listed here? In you experiments, did you find subsets of `FUSIBLE` useful?

Reply:
Every times the TF-XLA bridge support a new TF operation, we need to check if we want to support it by default or not. So we should document that somewhere to help this not to be forgotten. Any idea where we should document that?

I didn't used the shortcut much. What I used is FUSIBLE plus other operation that I wasn't sure if we want to include or not. But if someone want to start to play more with it, I think the shortcut would be useful.

To make this list, I passed over the TF operations that the bridge knows and selected those that I was sure what they do and that I was sure XLA could fuse them. I could have missed some. In 2 benchmarks, I found some that could maybe fused that I didn't include. I timed when including them and it slowed down XLALite compared to not having them. So I didn't included them. They where:
```   {"AssignVariableOp", "AssignAddVariableOp", "AssignSubVariableOp",
   "ReadVariableOp", "VarIsInitializedOp", "VariableShape",
   "ResourceApplyCenteredRMSProp", "ResourceApplyRMSProp",
   "ResourceScatterAdd", "ResourceApplyAdam"}```

Reply:
@sanjoy Can you recommend a good spot for this documentation? Do you have an opinion regarding the TF Ops Frederic did not include?

Reply:
As for the documentation I would vote for creating a unit test that checks that all non-whitelisted tf2xla ops are from a fixed set.  See [resource_operation_table_test.cc](https://github.com/tensorflow/tensorflow/blob/3039a4694e22674b502257ae34b0a5b614a631f3/tensorflow/compiler/tf2xla/resource_operation_table_test.cc#L1).

> Do you have an opinion regarding the TF Ops Frederic did not include?

Right it isn't totally obvious which ops were included.  I think we need some comments to describe the "format" of `whitelist_table`.

Reply:
Good idea to have a test that force to have all XLA/TF operations registered to be whiteliste or blacklisted. It will guaranty that we make a decision when new operations are supported.
I added this test.